bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -183,10 +246,10 @@
 
     <section class="fxos-community">
       <div class="content-wrapper">
-        <img src="{{ static('img/firefox/os-new/mozilla_groupshot.png') }}" width="260" height="260" alt="{{_('Mozilla community photo')}}" />
+        {{ high_res_img('img/firefox/os-new/mozilla-groupshot.png', {'alt': _('Mozilla community photo'), 'width': '260', 'height': '260'}) }}
```

Comment:
Should probably have empty alt here. This is essentially a presentational image rather than meaningful content.